Examples/Sample_xDscWebServiceRegistration.ps1

```diff
@@ -1,4 +1,4 @@
-# DSC configuration for Pull Server and Compliance Server
```

Comment:
There is an example to remove pull service that also needs to be updated:
Sample_xDscWebServiceRemoval.ps1